File: admin/src/api/DishesType.js
import axios from "axios";

const baseURL = "http://localhost:8080/api/dishesType";

//Dishes Type

export function getDishesType() {
  return axios.get(baseURL + "/getAllDishesType");
}

export function insertDishesType(name_Zh_HK, name_Zh_CN, name_Us_EN) {
  return axios.post(baseURL + "/insertDishesType", {
    name_Zh_HK,
    name_Zh_CN,
    name_Us_EN,
  });
}

export function editDishesType(name_Zh_HK, name_Zh_CN, name_Us_EN, id) {
  return axios.post(baseURL + "/editDishesType", {
    name_Zh_HK,
    name_Zh_CN,
    name_Us_EN,
    id,
  });
}

export function deleteDishesType(id) {
  return axios.post(baseURL + "/deleteDishesType", { id });
}

File: server/src/main/java/com/server/model/DishesType.java
package com.server.model;

import lombok.Data;

@Data
public class DishesType {

    private String id;
    private String name_Zh_HK;
    private String name_Zh_CN;
    private String name_Us_EN;

    @Override
    public String toString() {
        return "DishesType [name_Zh_HK=" + name_Zh_HK + ", name_Zh_CN=" + name_Zh_CN + ", name_Us_En="
                + name_Us_EN + "]";
    }
}
